projects/common/src/tokens/page-visibility.ts

```diff
@@ -13,7 +13,7 @@ export const PAGE_VISIBILITY = new InjectionToken<Observable<boolean>>(
                 startWith(0),
                 map(() => documentRef.visibilityState !== 'hidden'),
                 distinctUntilChanged(),
-                share(),
+                shareReplay({refCount: false, bufferSize: 1}),
```

Comment:
I wonder if it's enough to just move `share` up right under `fromEvent` 🤔

Reply:
Just tested it, and while that fixes the issue, we lose the benefits of the `share` operator (since everything in the pipe runs each time a subscriber gets an emission.

I propose using `shareReplay` to
1. Prevent functions executing `n` times (with `n` subscribers), vs only once per page visibility change with 'shareReplay'.
2. Keeping the most recent emission cached, so late subscribers always get that value.

Reply:
@waterplea @MarsiBarsi Just wanted to check back in and see if we could move this forwards.

Reply:
Right, sorry, slipped my radar. I'll get it merged and released today

Reply:
No problem, and thanks!